rcli/dispatcher.py

```diff
@@ -11,7 +11,8 @@
 import logging
 import sys
 import types  # noqa: F401 pylint: disable=unused-import
-import typing  # noqa: F401 pylint: disable=unused-import
+
+import typingplus as typing  # noqa: F401 pylint: disable=unused-import
```

Comment:
Move this down to be grouped with the external non-builtins. After colorama, or some such.